src/components/AssetsTable/index.jsx

```diff
@@ -68,16 +74,25 @@ export class AssetsTable extends React.Component {
       },
     };
 
+    this.deleteActions = [assetActions.DELETE_ASSET_FAILURE, assetActions.DELETE_ASSET_SUCCESS];
+
```

Comment:
Would it make sense to format these action types into delete/upload/lock/etc within the constants file [here](https://github.com/edx/studio-frontend/blob/master/src/data/constants/actionTypes.js#L3) instead? Something like:

`export const assetActions = {
  requestAssets: {
    REQUEST_ASSETS_SUCCESS: 'REQUEST_ASSETS_SUCCESS',
    REQUEST_ASSETS_FAILURE: 'REQUEST_ASSETS_FAILURE',
  },
  deleteAssets: {
    DELETE_ASSET_SUCCESS: 'DELETE_ASSET_SUCCESS',
    DELETE_ASSET_FAILURE: 'DELETE_ASSET_FAILURE',
  },
}`

And if we need to check if the assetType is a part of a subset (like we do [here](https://github.com/edx/studio-frontend/pull/93/files#diff-35e343ee67d70894218ba932e51bd1cdR412) ) then we won't have to replicate that data/update it if we add more types?

Reply:
I think that's a good idea. Would there be any reason to keep the constants file as-is (i.e. would categorizing the action types this way make it more difficult to access them for other components/files?).

Reply:
I'm on board with this

Reply:
I don't think it would be more difficult, but it may require additional updates in other files in this PR to handle the refactor correctly. The change would be - `assetActions.DELETE_ASSET_SUCCESS` to `assetActions.deleteAssets.DELETE_ASSET_SUCCESS` - so that would be more verbose / more to remember when referencing things / slightly duplicative.  Maybe breaking it apart by API type instead of by page? `deleteActions.DELETE_ASSET_SUCCESS`? We can hash this out tomorrow in person too.

Reply:
Also - We can merge this in as is with the Array (noting the Array.includes issue from below) and I'll create an immediate follow up branch to do the reactor of the action types so that this PR doesn't balloon.